lib/schemapack.js

```diff
@@ -207,11 +207,18 @@ class SchemaPack {
       return callback(e);
     }
 
+    collectionJSON = generatedStore.collection.toJSON();
+
+    // this needs to be deleted as even if version is not specified to sdk,
+    // it returns a version property with value set as undefined
+    // this fails validation against v2.1 collection schema definition.
+    delete collectionJSON.info.version;
```

Comment:
don't we need to check for the presence of the version prop? https://repl.it/repls/PleasingImpishVerification